crime-applications-adaptor/build.gradle

```diff
@@ -147,3 +148,17 @@ jsonSchema2Pojo {
 		}
 	}
 }
+
+spotless {
+	java {
+		target project.fileTree(project.rootDir) {
```

Comment:
I have a slight preference for the below with only explicit includes and no dependencies on the project attributes.

`target 'src/main/java/**/*.java', 'src/test/java/**/*.java'`

Reply:
Great shout - I'll update!